tooling/Microsoft.VisualStudio.RazorExtension/RazorPackage.vsct

```diff
@@ -13,7 +13,7 @@
         <Parent guid="guidSHLMainMenu" id="IDG_VS_WNDO_OTRWNDWS1"/>
         <Icon guid="guidImages" id="bmpPic1" />
         <Strings>
-          <ButtonText>RazorInfoToolWindow</ButtonText>
+          <ButtonText>Razor Diagnostics</ButtonText>
```

Comment:
This won't show up anywhere in what we're shipping. I just wanted to make it less nasty.